go/libraries/doltcore/sqle/dolt_index.go

```diff
@@ -255,7 +255,10 @@ func (di *doltIndex) keysToTuple(keys []interface{}) (types.Tuple, error) {
 	}
 	var vals []types.Value
 	for i, col := range di.cols {
-		val, err := col.TypeInfo.ConvertValueToNomsValue(keys[i])
+		// As an example, if our TypeInfo is Int8, we should not fail to create a tuple if given the value 9001.
+		// Instead, the index will return no values. Comparisons should happen at the widest type, only storage
```

Comment:
"return no values" isn't right... we could have a < comparison, in which case we return all values in the index.

Reply:
Good point, I'll update the comment.